README.md

```diff
@@ -29,12 +44,14 @@ will appear if the kernel ring buffer is overrun before messages are read. I try
 match [libevdev](https://www.freedesktop.org/software/libevdev/doc/latest/)
 closely, where possible.
 
-Writing to devices is not yet supported (eg, turning LEDs on).
```

Comment:
I removed this comment because the documentation indicates this is now supported. Is this correct?

Reply:
I think so, I'm still not actually sure what the "turning LEDS on" thing is but yes, stuff like uinputs can be written to.